avc/src/mapper/mapper.cpp

```diff
@@ -68,7 +68,7 @@ int main(int argc, char** argv) {
     pcl::VoxelGrid<pcl::PointXYZ> filter;
     filter.setLeafSize(0.2f, 0.2f, 0.2f);
 
-    ros::Rate rate(20);
+    ros::Rate rate(60);
```

Comment:
Mapper is publishing at 15hz on my machine whether the rate is set to 20 or 60, so I changed it back. I must have mistakenly suspected at some point that it could publish faster.